cluster/charts/rook-ceph/templates/deployment.yaml

```diff
@@ -207,6 +207,10 @@ spec:
         - name: ROOK_MON_OUT_TIMEOUT
           value: {{ .Values.mon.monOutTimeout }}
 {{- end }}
+{{- end }}
+{{- if .Values.unreachableNodeTolerationSeconds }}
+        - name: ROOK_UNREACHABLE_NODE_TOLERATION_SECONDS
+          value: {{ .Values.unreachableNodeTolerationSeconds }}
```

Comment:
For the integration test failure, you may need to add quotes around this value or else remove the quotes from the `"5"` in values.yaml.